tasks/main.yml

```diff
@@ -5,23 +5,113 @@
 
 # Examples of some tasks:
 - name: Ensure required packages are installed
-  package:
+  ansible.builtin.package:
     name: "{{ __aide_packages }}"
     state: present
     use: "{{ (__aide_is_ostree | d(false)) |
              ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+  when:
+    - aide_install
 
 - name: Ensure required services are enabled and started
-  service:
+  ansible.builtin.service:
     name: "{{ item }}"
     state: started
     enabled: true
   loop: "{{ __aide_services }}"
 
-- name: Generate /etc/{{ __aide_foo_config }}
-  template:
-    src: "{{ __aide_foo_config }}.j2"
-    dest: /etc/{{ __aide_foo_config }}
-    backup: true
+- name: Generate "/etc/{{ __aide_config }}"
+  ansible.builtin.template:
+#    remote_src: true
```

Comment:
Please add a variable like `aide_custom_template_remote_src` to use as a value for `remote_src` here. Sometimes it is useful to generate things from files on remote nodes.

Reply:
You're right, remote_src doesn't exist for template. Than this doesn't matter, thanks for pointing this out.